English/QUEST_JOBSTEP.tsv

```diff
@@ -3128,13 +3128,13 @@ QUEST_JOBSTEP_20150803_003127	Search for the traces of Lydia Schaffen
 QUEST_JOBSTEP_20150803_003129	Give them to the Fletcher Master
 QUEST_JOBSTEP_20150803_003130	You've found the bow of the first Fletcher Master. Return to the current Fletcher Master.
-QUEST_JOBSTEP_20150811_003131	Selamat datang. {nl}Kita kehabisan banyak hal, jadi kita harus memanfaatkan semua yang kita punya.
-QUEST_JOBSTEP_20150811_003132	Mothstem? Aku pernah melihatnya di Fedimian Suburbs.{nl}Pardoner Master juga sedang mencarinya. Apa kamu belum menemukannya?
-QUEST_JOBSTEP_20150811_003133	Cepat kembali ke Pardoner Master. Ia sedang mencari Jar of Fire itu sejak lama.
-QUEST_JOBSTEP_20150811_003134	Maksudmu Jar of Fire? Aku tahu. {nl}Tapi aku tidak akan memberitahumu secara cuma - cuma. Mungkin hardened oil dari Blue Puragi di Saknies Plains dapat membuatku berbicara...
-QUEST_JOBSTEP_20150811_003135	Bila kamu tidak membutuhkannya, tidak ada pilihan lain. {nl}Aku adalah seorang merchant, bukan sukarelawan.
-QUEST_JOBSTEP_20150811_003136	Oh, terima kasih banyak. {nl}Kudengar bahwa sesosok monster bernama Mothstem memiliki Jar of Fire dan aku takkan pernah melepaskannya.
+QUEST_JOBSTEP_20150811_003131	Welcome. {nl}We're running a lot of things, so we must take advantage of all we got.
+QUEST_JOBSTEP_20150811_003132	Mothstem? I've seen it in Fedimian Suburbs.{nl}Pardoner Master also looking for him. Did you not find them?
+QUEST_JOBSTEP_20150811_003133	Quickly returned to the Pardoner Master. He was looking for Jar of Fire long time ago.
+QUEST_JOBSTEP_20150811_003134	You mean Jar of Fire? I know. {nl}But I will not tell you for free. Perhaps hardened oil from the Blue Puragi in Saknies Plains can make me talk...
+QUEST_JOBSTEP_20150811_003135	If you do not need it, there is no other choice. {nl}I am a merchant, not a volunteer.
+QUEST_JOBSTEP_20150811_003136	Oh, thank you very much. {nl}I heard that a monster named Mothstem have Jar of Fire and I'll never let go.
 QUEST_JOBSTEP_20150811_003137	Um, aku dengar bahwa Item Merchant melihatnya belum lama ini. Tanyakan padanya tentang hal itu.
-QUEST_JOBSTEP_20150811_003138	Ini bencana. Sangat mengecewakan. {nl}Seorang merchant sepertiku tidak sepenuhnya paham apa yang kamu rasakan tapi tetap saja...
-QUEST_JOBSTEP_20150811_003139	Untuk semua itu, bantulah aku dan aku akan memberimu sesuatu. Mungkin tidak sebagus yang kamu rusakkan tapi mungkin akan berguna.
-QUEST_JOBSTEP_20150811_003140	Aku tidak meminta banyak. {nl}Kamu dapat melihat ini dalam tanda - tanda yang dibuat para pertapa di sepanjang Starving Demon's Way.
+QUEST_JOBSTEP_20150811_003138	This is a disaster. Very disappointing. {nl}A merchant like me do not fully understand how you feel but still...
+QUEST_JOBSTEP_20150811_003139	So, help me and I'll give you something. Maybe not as good as you shred but might be useful.
+QUEST_JOBSTEP_20150811_003140	I do not ask much. {nl}You can see this in the sign made by the hermits along Starving Demon's Way.
```

Comment:
> _Kamu dapat melihat ini dalam tanda - tanda (...)_

karena kata tanda nya jamak, jadi _You can see this in the **signs**_ (kurang 's')